app/controllers/facilities_controller.rb

```diff
@@ -68,9 +68,13 @@ def list
     @active_tab = "manage_facilites"
     if session_user.administrator?
       @facilities = Facility.alphabetized
```

Comment:
It looks like we're not using `@facilities` in the view any more, is that right?
If so let's change it to a local variable `facilities`